tdrs-backend/manifest.yml

```diff
@@ -4,5 +4,6 @@ applications:
   memory: 256M
   instances: 1
   disk_quota: 2G
```

Comment:
cloud foundry can automaticly detect the build pack, but we felt it was best to be explicit that we are using the python build pack to avoid any uncertainty or possibility that the detection mechanisim might misbehave.